ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AttestationManager.java

```diff
@@ -221,6 +236,7 @@ private void sendToSubscribersIfProducedLocally(ValidateableAttestation attestat
     }
 
     notifyAttestationsToSendSubscribers(attestation);
+    notifyAllValidAttestationsSubscribers(attestation);
```

Comment:
one thing we should do is update the open api from the events to remove the statement that we dont support attestation and blocks...

Reply:
I was planning to update the changelog and do all the cleaning when I finished the voluntary_exit as well.

Reply:
ok all good.